docs/utilities/permissionapi.md

```diff
@@ -0,0 +1,63 @@
+PermissionAPI
+=============
+
+The PermissionAPI is a very basic implementation of a permission system.
+
+Its default implementation doesn't add an advanced permission handling (like we know it for example from PEX), 
+
+but instead it has 3 permission levels, (ALL = all players, OP = operators, NONE = neither normal players or operators).
+
+This behaviour can be changed by mods which implement an own PermissionHandler.
+
+How to use the PermissionAPI
+-----------------------------
+
+For basic support you just need to call `PermissionAPI.hasPermission(EntityPlayer player, String  node)`,
+
+though by default this is going to return always false, as the default implementation uses the permission level `NONE`
+
+So if we want that all players, or just OP's to be able to use this  we also need to register our permission node.
+
+Achieving this is as simple as checking for permissions: `PermissionAPI.registerNode(String node, DefaultPermissionLevel level, String description)`, 
+
+though this has to be done in Init or Later.
+
+!!! note
+    The PermissionAPI isn't restricted to be used for commands, you could also use it for other things, like restricting access to a GUI.
+    You do also need to check if your CommandSender is a player if you use it in combination with commands!
+
+DefaultPermissionLevel
+--------------
+
+The DefaultPermissionLevel has 3 Values:
+
+Permission node
+---------------------------------------
+
+While there are theoretically no rules for the permission nodes, the best practice for them is to be `modid.subgroup.permission_id`
+
+It is recommended to use this naming scheme as other implementations may have stricter rules.
+
+Making an own Implementation of the PermissionHandler
```

Comment:
an own -> you rown
Implementation -> implementation
PermissionHandler -> `PermissionHandler`